flask_hw/flask_intro/app.py

```diff
@@ -0,0 +1,25 @@
+from flask import Flask, render_template
+
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def get_home():
+    return render_template('home.html')
+
+
+@app.route('/vegetables')
+def get_vegetables():
+    vegetables = ['beans', 'carrot', 'beetroot', 'cucumber']
+    return render_template('vegetables.html', vegan_list=vegetables)
+
+
+@app.route('/fruits')
+def get_fruits():
+    fruits = ['melon', 'apple', 'strawberry', 'grape']
```

Comment:
U may use tuple instead list. it is not critical here. But if you see that array will not changed, use tuple

Reply:
fixed